app/assets/stylesheets/govuk_publishing_components/components/_lead-paragraph.scss

```diff
@@ -12,3 +12,13 @@
 .gem-c-lead-paragraph--inverse {
   color: govuk-colour("white");
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .gem-c-lead-paragraph {
+    break-inside: avoid;
```

Comment:
Nitpick: I wonder if this is necessary - the lead paragraph generally occurs at the start of content, so it's unlikely to be at the boundary of a page, plus even if it does overlap the bottom of a page, forcing it to start on a new page might actually end up using more paper than necessary - thoughts?

Reply:
Good point, I've removed it and also the `!important` and the stylelint exclusion for important.